components/org.wso2.carbon.identity.application.authenticator.hypr.rest/org.wso2.carbon.identity.application.authenticator.hypr.rest.common/src/main/java/org/wso2/carbon/identity/application/authenticator/hypr/rest/common/Util.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.authenticator.hypr.rest.common;
+
+import org.slf4j.MDC;
+import org.wso2.carbon.identity.application.authenticator.hypr.common.constants.HyprAuthenticatorConstants;
+
+import java.util.UUID;
+
+/**
+ * Util class.
+ */
+public class Util {
+
+    /**
+     * Constructor for Util class.
+     */
+    private Util() {}
```

Comment:
Do wee need this?

Reply:
Fixed with [eb39195](https://github.com/wso2-extensions/identity-outbound-auth-hypr/pull/1/commits/eb39195b388a647223846891193117538a081295)